simulator-spring-boot/src/main/java/org/citrusframework/simulator/listener/SimulatorStatusListener.java

```diff
@@ -116,9 +120,9 @@ public void onTestActionStart(TestCase testCase, TestAction testAction) {
         if (!ignoreTestAction(testAction)) {
             if (logger.isDebugEnabled()) {
                 logger.debug(testCase.getName() + "(" +
-                        StringUtils.arrayToCommaDelimitedString(getParameters(testCase)) + ") - " +
-                        testAction.getName() + ": " +
-                        (testAction instanceof Described && StringUtils.hasText(((Described) testAction).getDescription()) ? ((Described) testAction).getDescription() : ""));
+                    arrayToCommaDelimitedString(getParameters(testCase)) + ") - " +
+                    testAction.getName() + ": " +
```

Comment:
```suggestion
                                        testAction.getName() + 
                    (testAction instanceof Described described && hasText(described.getDescription()) ? ": " + described.getDescription() : ""));
```

Reply:
sorry, but what is the implied change? 🐙

Reply:
Just a cosmetic issue: if the description is empty, the trailing colon could be removed by this change.

Reply:
ah, now I get it. I think you've marked the wrong line 😉